client-ui/src/app/components/execution-plan-details/execution-plan-details.component.spec.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ExecutionPlanDetailsComponent } from './execution-plan-details.component';
+
+describe('ExecutionPlanDetailsComponent', () => {
+  let component: ExecutionPlanDetailsComponent;
+  let fixture: ComponentFixture<ExecutionPlanDetailsComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ExecutionPlanDetailsComponent]
+    })
+      .compileComponents();
+  }));
```

Comment:
**nit: indents**
In such cases I usually leave the first object on the line alone (or on the same line with the method definition for one-liner methods). It almost always help the auto layout to make the indents nicer.
E.g.
```suggestion
  beforeEach(async(() => TestBed
    .configureTestingModule({
      declarations: [ExecutionPlanDetailsComponent]
    })
    .compileComponents();
```
or even 
```suggestion
TestBed
    .configureTestingModule({ declarations: [ExecutionPlanDetailsComponent] })
    .compileComponents();
```